.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -72,6 +72,7 @@ ATest
 atid
 ation
 atlassian
+atn
```

Comment:
We are going to need to add these for the upgrade to ATNLR4 anyway, I don't think we will be able to get around it.